lib/routes/users.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Controller for Users
+ *
+ * @module
+ */
+
+'use strict';
+
+const _ = require('lodash'),
+  responses = require('../responses'),
+  controller = require('../services/users');
+
+/**
+ * All routes go here.
+ *
+ * They will all have the form (req, res), but never with next()
+ *
+ * @namespace
+ */
+let route = _.bindAll({
+  /**
+   * @param {object} req
+   * @param {object} res
+   */
+  post(req, res) {
+    responses.expectJSON(function () {
+      return controller.post(req.uri, req.body, res.locals);
+    }, res);
+  }
+}, [
+  'post'
+]);
+
+function routes(router) {
+  router.use(responses.varyWithoutExtension({varyBy: ['Accept']}));
+
+  // list users
+  router.all('/', responses.methodNotAllowed({allow: ['get', 'post']}));
+  router.all('/', responses.notAcceptable({accept: ['application/json']}));
+  router.get('/', responses.list());
```

Comment:
mmmm yeah wait what? Can you elaborate on what's being exposed through this GET? Definitely seems weird to have this publicly available (especailly if it includes providers and usernames?)